src/components/new-members/index.js

```diff
@@ -28,7 +55,7 @@ const NewMemberList = ({ isOptionKey }) => {
       <div className={styles.container}>
         {filterMembers.map((newMember) => (
           <React.Fragment key={newMember.id}>
-            {renderNewMember(newMember, isOptionKey)}
+            {renderNewUsers(newMember, isOptionKey, isSuperUser)}
```

Comment:
Why not let the `renderNewUsers` deal with the super user context itself, instead of passing it as an argument?

Reply:
Maybe it should handle the Option key within `renderNewUsers` too?

Reply:
Also, I feel it should be called `renderNewUser` (Singular)

Reply:
> Why not let the `renderNewUsers` deal with the super user context itself, instead of passing it as an argument?

I thought instead of fetching from the context for every new user. it would be better if I get `isSuperUser` from context once and pass it to each new user card component. Let me know which approach is more optimized. I will change accordingly.

Reply:
> Also, I feel it should be called `renderNewUser` (Singular)

yeah Right. My Bad. I will change it.

Reply:
> Maybe it should handle the Option key within `renderNewUsers` too?

since we get isOptionKey as props and not from context. We need to pass it to `renderNewUsers`

Reply:
>I thought instead of fetching from the context for every new user. it would be better if I get isSuperUser from context once and pass it to each new user card component. Let me know which approach is more optimized. I will change accordingly.
Do not pass the argument, let the function handle that please.

Reply:
> > I thought instead of fetching from the context for every new user. it would be better if I get isSuperUser from context once and pass it to each new user card component. Let me know which approach is more optimized. I will change accordingly.
> 
> Do not pass the argument, let the function handle that please.

Sure @sumitd94 . May I also know the reason behind this? Tried to google but couldn't get the answer 🥲